alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/CustomMessageConfigModel.java

```diff
@@ -28,38 +28,48 @@
 
 import com.synopsys.integration.alert.common.persistence.accessor.FieldAccessor;
 
-public class TestConfigModel extends AlertSerializableModel {
+public class CustomMessageConfigModel extends JobFieldModel {
     private final String destination;
     private final FieldAccessor fieldAccessor;
-    private String configId;
```

Comment:
configId was useful here because more than just jobs can be tested. Now that tests use this model, we have to reference this value through jobId which will be confusing.